modules/mobile/spec/models/appointment_spec.rb

```diff
@@ -3,17 +3,13 @@
 require 'rails_helper'
 
 RSpec.describe Mobile::V0::Appointment, type: :model do
-  describe '.toggle_non_prod_id!' do
-    it 'toggles mocked ids to real ones' do
-      expect(Mobile::V0::Appointment.toggle_non_prod_id!('983')).to eq('442')
-    end
-
-    it 'toggles real ids back to mocked ones' do
-      expect(Mobile::V0::Appointment.toggle_non_prod_id!('442')).to eq('983')
+  describe '.convert_non_prod_id!' do
```

Comment:
I think we could also use a test for "passes through other ids". It might also be worth stubbing the rails env to confirm that it behaves differently in production (assuming I'm reading the intent correctly).

Reply:
Works for me. Added a couple specs